src/test/java/io/xstefank/wildfly/bot/PRCommitCheckTest.java

```diff
@@ -184,4 +188,23 @@ public void testOverridingCommitMessage() throws IOException {
                     Util.verifyFailedFormatComment(mocks, pullRequestJson, "- Lorem ipsum dolor sit amet");
                 });
     }
+
+    @Test
+    public void testDisableGlobalFormatCheck() throws IOException {
+        wildflyConfigFile = """
+                wildfly:
+                  format:
+                    enabled: false
+                """;
+        mockedContext = MockedContext.builder(pullRequestJson.id())
+                .commit(INVALID_COMMIT_MESSAGE);
+        given().github(mocks -> Util.mockRepo(mocks, wildflyConfigFile, pullRequestJson))
+                .when().payloadFromString(pullRequestJson.jsonString())
+                .event(GHEvent.PULL_REQUEST)
+                .then().github(mocks -> {
+                    GHRepository repo = mocks.repository(TEST_REPO);
+                    Mockito.verify(repo, never()).createCommitStatus(anyString(), any(), anyString(), anyString());
+                    Mockito.verify(repo, never()).createCommitStatus(anyString(), any(), anyString(), anyString(), anyString());
```

Comment:
probably doesn't need to verify "never" two times

Reply:
I verify that neither of the methods is called, as it's overloaded method

Reply:
ok, I missed the number of arguments. ok then